be1-go/message/messagedata/chirp_delete.go

```diff
@@ -1,9 +1,29 @@
 package messagedata
 
+import (
+	"encoding/base64"
+	"golang.org/x/xerrors"
+)
+
 // ChirpDelete defines a message data
 type ChirpDelete struct {
 	Object    string `json:"object"`
 	Action    string `json:"action"`
 	ChirpId   string `json:"chirp_id"`
-	Timestamp int    `json:"timestamp"`
+	Timestamp int64  `json:"timestamp"`
+}
+
+// Verify verifies that the ChirpDelete message is correct
+func (message ChirpDelete) Verify() error {
+	// verify that Timestamp is positive
+	if message.Timestamp < 0 {
+		return xerrors.Errorf("timestamp is %d, should be minimum 0", message.Timestamp)
+	}
+
+	// verify that the chirp id is base64URL encoded
+	if _, err := base64.URLEncoding.DecodeString(message.ChirpId); err != nil {
```

Comment:
we don't use assignation in if expression because it makes the code less readable. Please do it in two steps.

Reply:
same above

Reply:
okay, I've changed this for all the verify() functions 👍